doc/deprecated.rst

```diff
@@ -32,6 +32,12 @@ Extensions
   After:
   $twig->getRuntime(EscaperRuntime::class)->METHOD();
 
+Nodes
+-----
+
+* The ``Twig\Node\Expression\CallExpression::compileArguments()`` method second
```

Comment:
`The second argument of the ``Twig\Node\Expression\CallExpression::compileArguments()`` method is deprecated`